Misc/NEWS.d/next/Library/2022-02-05-22-14-44.bpo-46644.P--1Cz.rst

```diff
@@ -0,0 +1 @@
+No longer require valid typeforms to be callable. Patch by Gregory Beauregard.
```

Comment:
This is a bit hard to understand for a user who isn't familiar with the internals of the typing module. Maybe list out a few of the bugs it fixes, such as Annotated over InitVar and ParamSpecArgs.

Reply:
Ah, I was targeting users trying to implement their own typeforms with the wording, but that's a good point. I added it.